npu.user.js

```diff
@@ -7,6 +7,7 @@
 // @include        https://*hallgato*.*neptun*/*
 // @include        https://netw*.nnet.sze.hu/hallgato/*
 // @include        https://nappw.dfad.duf.hu/hallgato/*
+// @include        https://neptun*.uni-bge.hu/hallgato/*

```

Comment:
First `@include` line above already covers this. NPU is active when I go to the BGE Neptun login page.
https://github.com/solymosi/npu/pull/4/files#diff-d04c3ea501de8700838acf78a61ba87cR6

Is there a scenario where this is not the case?